code/game/objects/effects/decals/Cleanable/humans.dm

```diff
@@ -148,10 +148,10 @@ var/global/list/blood_list = list()
 
 
 
-/obj/effect/decal/cleanable/blood/gibs/proc/streak(var/list/directions)
+/obj/effect/decal/cleanable/blood/gibs/proc/streak(var/list/directions, spread_radius = 0)
 	spawn (0)
 		var/direction = pick(directions)
-		for (var/i = 0, i < pick(1, 200; 2, 150; 3, 50; 4), i++)
+		for (var/i = 0 to spread_radius)
```

Comment:
So your gibs are now stuck to you unless you're injected with spacelube? Seems pretty crummy.